project.clj

```diff
@@ -8,32 +12,77 @@
 
   :pedantic? :abort
 
-  :dependencies [[org.jruby/jruby-core "1.7.26"
-                  :exclusions  [com.github.jnr/jffi
-                                com.github.jnr/jnr-x86asm
-                                joda-time
-                                org.yaml/snakeyaml]]
+  ;; Note that there is a fair amount of duplication between the
```

Comment:
Would it make sense to define a vector of all the asm dependencies that could be reused? or would that be too complicated/hard to read given this is all a macro?